src/NUnitFramework/framework/Internal/Execution/CompositeWorkItem.cs

```diff
@@ -231,13 +240,52 @@ private void RunChildren()
 
         private void CreateChildWorkItems()
         {
-            _children = new List<WorkItem>();
+            Children = new List<WorkItem>();
 
             foreach (ITest test in _suite.Tests)
```

Comment:
Now that this foreach is getting longer, can you add brackets so that the scope is clear. We only use this form for short blocks like it was before.